### DISCUSSION TITLE
5.0 release Attributes

### DISCUSSION THREAD

**Discussion by sharathkashyap** [Sunbird-Ed/Community#68] [General Discussion]
Inviting all the stakeholders to effectively help us to come up with planned release scope for 5.0 from all building blocks.

Sunbird ED @luckynvdu @sharathkashyap 
Sunbird Knowlg @Krishnaj20 @vinukumar-vs
Sunbird Inquiry @Krishnaj20 @pallakartheekreddy
Sunbird CoKreate @Krishnaj20 @pallakartheekreddy
Sunbird Lern @rhwarrier @reshmi-nair
Sunbird Observ @rhwarrier @anandp504

DC's: @SanthoshVasabhaktula @vrayulu
Sunbird QA: @ashokreddy1208
@gandham-santhosh
@kameshbhr @mmanimegalai
@madhucr @alok-os
@anandp504 @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @ashokreddy1208 @kameshbhr @mmanimegalai 

https://app.gitbook.com/o/-Mi9QwJlsfb7xuxTBc0J/s/-MkgPDmvKwE_DgYJbvPS/use/releases/planned-items/4.10.1-4.10.2-4.10.3-and-5.0-major

Kindly provide the links in the same discussion thread. This helps us to have a unified view of deliveries from all building blocks for the release.

cc: @chitranshu-keshav @luckynvdu

**Comment by vinukumar-vs** [comment]
@sharathkashyap @luckynvdu 
Please refer the latest release notes of Knowlg 
https://knowlg.sunbird.org/use/release-notes/latest

Going forward also please refer to the same link for any knowledge releases. 
@rhwarrier

**Comment by ashokreddy1208** [comment]
Hi All, Can you please share planned release scope for 5.0 from all building blocks as we need to share with DIKSHA Adopter.

Sunbird ED @luckynvdu @sharathkashyap
Sunbird Knowlg @rhwarrier  @vinukumar-vs
Sunbird Inquiry @surendrasinghs  @pallakartheekreddy
Sunbird CoKreate @surendrasinghs  @pallakartheekreddy
Sunbird Lern @rhwarrier @reshmi-nair
Sunbird Observ @rhwarrier @anandp504

**Comment by rhwarrier** [comment]
The roadmap page for Lern and Obsrv has the scope (and the linked tickets) for 5.0
https://lern.sunbird.org/learn/product-roadmap
https://obsrv.sunbird.org/learn/product-roadmap

**Comment by luckynvdu** [comment]
@rhwarrier @surendrasinghs @vinukumar-vs @pallakartheekreddy requesting to share the scope for 5.0 for Knowlg, CoKreat, and Inquiry as well

**Comment by surendrasinghs** [comment]
For inQuiry, please refer to https://inquiry.sunbird.org/use/release-notes
For coKreat, please refer to  and https://cokreat.sunbird.org/use/release-notes and https://project-sunbird.atlassian.net/jira/software/c/projects/CO/boards/107/backlog?issueLimit=100

**Comment by vaibhavbhuva** [comment]
@sharathkashyap @vinukumar-vs 
New tags have been updated for the CSP changes.

Please refer the latest release notes of Knowlg
https://knowlg.sunbird.org/use/release-notes/latest

Video streaming & GCP changes (content-service) of pre-signed URL is still progress. Will update once tags created.

@rhwarrier @luckynvdu

**Comment by rajeevsathish** [comment]
@vinukumar-vs @surendrasinghs @rhwarrier @pallakartheekreddy 
if any player or editor version is changed that has to be used by SunbirdED please mention
also any Backend API that has got changed that has to be updated or used by SunbirdED please mention the same.
cc: @sharathkashyap  @luckynvdu @swayangjit

**Reply by vinukumar-vs** [reply]
@rajeevsathish All the details are documented in the release notes itself.
@sharathkashyap  @swayangjit
